tests/common/fixtures/consistency_checker/constants.py

```diff
@@ -1,6 +1,31 @@
+from dataclasses import dataclass
+from typing import List
+
+ALL_ATTRIBUTES = "all"
+
+
+@dataclass
+class ConsistencyCheckQueryKey:
+    key: str
+    attributes: List[str]
+
+
+ARISTA_KEYS: List[ConsistencyCheckQueryKey] = [
```

Comment:
Non-blocking comment, it may be better to name these variable based on the specific ASIC, like `BROADCOM_TH_KEYS` and `BROADCOM_TH2_KEYS`. That'll make it clear/reusable for other platforms in the future.